share/eds/store.go

```diff
@@ -284,7 +290,7 @@ func (s *Store) put(ctx context.Context, root share.DataHash, square *rsmt2d.Ext
 	go func() {
 		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 		defer cancel()
-		_, err := s.cache.GetOrLoad(ctx, result.Key, s.getAccessor)
+		_, err := s.cache.First().GetOrLoad(ctx, result.Key, s.getAccessor)
```

Comment:
comment above this go func can be modified

Reply:
Could you please elaborate in what way it needs to be modified? It refers to the fact, that not any block, that is put into the store is actually recent, because it could be historical sampling block

Reply:
Nevermind, I read it wrong